scipy/special/add_newdocs.py

```diff
@@ -3036,10 +3036,86 @@ def add_newdoc(place, name, doc):
     """)
 
 add_newdoc("scipy.special", "hyp2f1",
-    """
+    r"""
     hyp2f1(a, b, c, z)
 
-    Gauss hypergeometric function 2F1(a, b; c; z).
+    Gauss hypergeometric function 2F1(a, b; c; z), defined as:
+
```

Comment:
This needs a ``.. math::`` directive above it surrounded by blank lines. And then indent the LaTeX.